cmd/mqtt/main.go

```diff
@@ -135,6 +138,24 @@ func main() {
 	}
 	defer nps.Close()
 
+	if cfg.mqttTargetHealthCheck != "" {
+		for {
+			res, err := http.Head(cfg.mqttTargetHealthCheck)
+			if err != nil {
+				logger.Info(fmt.Sprintf("Broker not ready: %s ", err.Error()))
+				time.Sleep(1 * time.Second)
+				continue
+			}
+			if res.StatusCode != 200 {
```

Comment:
Do not use `200`, there are [status code constants](https://golang.org/src/net/http/status.go)